FlowCrypt/Models/Realm Models/RecipientRealmObject.swift

```diff
@@ -22,23 +22,23 @@ extension RecipientRealmObject {
         name: String?,
         lastUsed: Date?,
         keys: [PubKey]
-    ) {
+    ) throws {
         self.init()
         self.email = email
         self.name = name ?? ""
         self.lastUsed = lastUsed
 
-        let realmKeys = keys
-            .compactMap { try? PubKeyRealmObject($0) }
+        let realmKeys = try keys
+            .compactMap { try PubKeyRealmObject($0) }
```

Comment:
In this case recipient saving won't succeed even when some keys are valid. Previous code filtered `realmKeys` to include only keys with valid `primaryFingerprint`.

I think it'll be better to replace `try?` with optional init, so `PubKeyRealmObject` will return `nil` when `fingerprints` are empty.